auth-web/src/util/constants.ts

```diff
@@ -165,14 +165,40 @@ export enum LearFilingTypes {
 // see also https://github.com/bcgov/bcrs-shared-components/blob/main/src/modules/corp-type-module/corp-type-module.ts
 export enum CorpTypes {
     // actual corp types
+
+    // in colin but modernized
     BC_CCC = 'CC',
     BC_COMPANY = 'BC',
     BC_ULC_COMPANY = 'ULC',
+
+    // modernized
     BENEFIT_COMPANY = 'BEN',
     COOP = 'CP',
     PARTNERSHIP = 'GP',
     SOLE_PROP = 'SP',
 
+    // colin
+    CCC_CONTINUE_IN = 'CCC',
+    BC_CORPORATION = 'CR', // SPECIAL NAMEREQUEST-ONLY ENTITY TYPE
+    ULC_CONTINUE_IN = 'CUL',
+    EXTRA_PRO_REG = 'EPR',
+    LL_PARTNERSHIP = 'LL',
+    LIMITED_CO = 'LLC',
+    LIM_PARTNERSHIP = 'LP',
+    BC_UNLIMITED = 'UL', // SPECIAL NAMEREQUEST-ONLY ENTITY TYPE
+    XPRO_LL_PARTNR = 'XL',
+    XPRO_LIM_PARTNR = 'XP',
+
+    // others
+    FINANCIAL = 'FI',
+    PRIVATE_ACT = 'PA',
+    PARISHES = 'PAR',
+
+    // societies
+    CONT_IN_SOCIETY = 'CS',
+    SOCIETY = 'S',
+    XPRO_SOCIETY = 'XS',
```

Comment:
This list is a duplicate of https://github.com/bcgov/bcrs-shared-components/blob/main/src/modules/corp-type-module/corp-type-module.ts and it would be best to not have both.

I realize changing over to using the shared enum will take some work. I'm thinking we accept the change above and create a tech debt ticket to fix this later. @seeker25 , do you have a strong preference for changing this now instead?

Reply:
Rather not block - we can deal with it later

Reply:
Ok with me. Hongjing, please create a tech debt ticket for this.

Reply:
Hongjing, I'll create the new ticket, so you can focus on the other comments.

Reply:
Wait... `CorpTypes` has 3 overloaded values (TMP, NR, RTMP) which `CorpTypeCd` doesn't have. Is it good to add them to `CorpTypeCd`?

Reply:
No, but that's a problem for the other ticket :)

(It might be possible to extend an enum or something. Or fix the code to not use overloaded values...)